### DISCUSSION TITLE
How to write image to stdout

### DISCUSSION THREAD

**Discussion by daryl-williams** [dlemstra/Magick.NET#1418] [Help]
Hi Folks,

I'm new to dotnet and Magick.NET and trying to write a command line tool that can read and write from/to stdin and stdout, but haven't been able to find info on how to write to stdout. Any help, rtfm's, etc. would be much appreciated. 

Thanks,

Daryl

**Comment by dlemstra** [comment]
You could use `Console.OpenStandardOutput()` and write to that stream?
